### ISSUE TITLE
Incorrect font height

### ISSUE THREAD

**Issue by INPUTsys-Chris** [vandeseer/easytable#143]
Font height is determined in PdfUtils. The code corresponds to the 'best' SO answer if one googles 'PDFBox font height'.
 
But that codes uses getCapHeight and is wrong if I have descents or Umlauts. Cap Height is just the height of capital letters.

```
for (int fontSize : new int[]{6, 10, 12, 24, 48}) {
    tableBuilder.addRow(
            Row.builder()
                    .add(TextCell.builder()
                                 .text("gÜ Text with word wrap")
                                 .padding(0f)
                                 .borderWidth(0.000001f)
                                 .fontSize(fontSize)
                                 .build())
                    .build());
}
```

![grafik](https://user-images.githubusercontent.com/56319603/170214664-235b1039-bd30-4e86-b7d4-3a6271e65516.png)

My proposal would be to determine the height this way:

```
PDFontDescriptor fd = font.getFontDescriptor();
return (fd.getFontBoundingBox().getHeight() - fd.getLeading()) / 1000.0f * fontSize;
```

**Comment by vandeseer** [comment]
Thanks for your suggestion @INPUTsys-Chris! 
I will check that (as soon as I find some time for it).

**Comment by vandeseer** [comment]
I added the correction and introduced a flag to allow for backwards compatibility. Would be nice if you could have a look @INPUTsys-Chris and try it out. Still needs some tests IMHO ...

**Comment by vandeseer** [comment]
This is the PR: https://github.com/vandeseer/easytable/pull/149

**Comment by INPUTsys-Chris** [comment]
Thank you for your hard work.
I ran my quick test and while the spacing is better, the letters don't seem to be on the right place:

![grafik](https://user-images.githubusercontent.com/56319603/185064664-04ddd450-a0e6-4b28-83ef-c0820fa61510.png)

**Comment by vandeseer** [comment]
Yeah, there's clearly something odd here. Thanks. If I will have a look when I find some time.
